### ISSUE TITLE
[3.7.0+] [com_fields] modal untranslated strings

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#15521]
### Steps to reproduce the issue

install 3.7.0 
create some custom fields
go to frontend and login as a user
fake editing an article and clicking the field button in tinyMCE by just loading the URL (with a valid token in the url):

http://127.0.0.1:8000/index.php?option=com_fields&view=fields&layout=modal&tmpl=component&context=com_content.article&editor=jform_articletext&98fec83879efc9950cec2daaea1da68f=1

in a WIDE MONITOR so that the ORDERING and ORDER LIMIT dropdowns are shown.

Note that in normal use this url is loaded in a modal window and css breakpoints HIDE the ordering/limit dropdowns.

### Expected result

That all text strings are translated

Either
1) Translate the strings
or
2) Remove the dropdowns that are being hidden by breakpoints in screen width caused by modal

### Actual result

<img width="318" alt="screen shot 2017-04-24 at 21 58 01" src="https://cloud.githubusercontent.com/assets/400092/25358371/1d00c39c-2939-11e7-8985-d9708cc14e96.png">
<img width="1437" alt="screen shot 2017-04-24 at 21 57 52" src="https://cloud.githubusercontent.com/assets/400092/25358372/1d02aa0e-2939-11e7-937b-caa592828724.png">

**Comment by infograf768** [comment]
So, if I understand well, you are forcing loading in a normal window what should be in a modal .
I would not consider this as a bug.

**Comment by PhilETaylor** [comment]
I disagree, anything that is rendered out in the DOM of a page should be correctly translated, even if that element is hidden by css or screen size breakpoints.

3pd are well known for reusing "core" features and so if a 3pd reused this view in a different way the width of a modal might be different, and show these useful filters, and they would be untranslated.

The un translated strings would also be logged during debugging

**Comment by infograf768** [comment]
Well, I then guess it is just a matter of adding these strings in frontend.

**Comment by infograf768** [comment]
Tested
Debug language displays in this case
```
# JROOT/administrator/components/com_fields/helpers/fields.php

JGLOBAL_FIELD_GROUPS="FIELD GROUPS"

# JROOT/libraries/joomla/form/fields/list.php

JGLOBAL_SORT_BY="SORT BY"
JGLOBAL_TITLE_ASC="TITLE ASC"
JGLOBAL_TITLE_DESC="TITLE DESC"
JGRID_HEADING_ACCESS_ASC="HEADING ACCESS ASC"
JGRID_HEADING_ACCESS_DESC="HEADING ACCESS DESC"
JGRID_HEADING_ID_ASC="HEADING ID ASC"
JGRID_HEADING_ID_DESC="HEADING ID DESC"
JGRID_HEADING_LANGUAGE_ASC="HEADING LANGUAGE ASC"
JGRID_HEADING_LANGUAGE_DESC="HEADING LANGUAGE DESC"
JGRID_HEADING_ORDERING_ASC="HEADING ORDERING ASC"
JGRID_HEADING_ORDERING_DESC="HEADING ORDERING DESC"
JSTATUS_ASC="ASC"
JSTATUS_DESC="DESC"

# JROOT/libraries/cms/layout/file.php

JTOGGLE_HIDE_SIDEBAR="HIDE SIDEBAR"
JTOGGLE_SHOW_SIDEBAR="SHOW SIDEBAR"
```
but it gets worse as "Field Group" is displayed 3 times in the dropdown.
Therefore, just adding the strings may not be the correct way to do.

**Comment by brianteeman** [comment]
Closed as there is a PR #17433
